gradle.properties

```diff
@@ -13,3 +13,4 @@ avito.git.state=local
 # Without it failed on studio sync when ci=true. It happens because studio makes eager configuration of each register task.
 avito.repo.ssh.url=
 useCompositeBuild=true
+avito.build-verdict.enabled=false
```

Comment:
What about an extension instead of properties?
It'll be more convenient for clients

Reply:
Example:
```kotlin
buildVerdict {
   enabled = myCustomFunction()
}
```

Reply:
Agree with that in common cases extension is preferable then property, but for the current case, the extension is an overhead. Only we and only once need to enable it on CI and disable for local purposes. So property will be enough